create-envfile.py

```diff
@@ -0,0 +1,142 @@
+import argparse
+import json
+import logging
+import os
+import random
+import re
+import string
+import sys
+from datetime import datetime
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+logger = logging.getLogger()
+handler = logging.StreamHandler(sys.stdout)
+logger.setLevel(logging.INFO)
+formatter = logging.Formatter("%(levelname)s - %(message)s")
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+
+def shuffle(chars):
+	chars_as_list = list(chars)
+	random.shuffle(chars_as_list)
+	return ''.join(chars_as_list)
+
+_simple_chars = shuffle(string.ascii_letters + string.digits)
+_strong_chars = shuffle(string.ascii_letters + \
+    string.digits + \
+    string.punctuation.replace("\"", '').replace("'", '').replace("`", ''))
+
+
+def generate_env_file(parser):
+    ################################################################
+    #######                     Required                    ########
+    ################################################################
+    parser.add_argument("hostname", help="host name")
```

Comment:
I would add a default value here `localhost`

Reply:
Was requested to be a required parameter, providing a default makes it an optional param

Reply:
it was requested to be required, but I agree with @afabiani.

Reply:
Done 👍